### PR TITLE
Fix regression errors for arm64 nightlies.

### PR DESCRIPTION
**Pull request by mpreiner**

### CODE REVIEW

File: test/regress/regress0/smtlib/set-info-status.smt2 [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ; EXPECT: sat
 ; EXPECT: unsat
 ; EXPECT-ERROR: Expected result unsat but got sat
-; ERROR-SCRUBBER: sed -e '/Fatal failure within.*/d'
+; ERROR-SCRUBBER: sed -e '/Fatal failure within.*/d' -e '/qemu: uncaught target signal.*/d'
```
Comment (nafur) [inline test/regress/regress0/smtlib/set-info-status.smt2:6]: Could we move that into the general regression script? This is not something specific to this regression, but would always occur when exit code is non-zero, right?

Comment (mpreiner) [reply test/regress/regress0/smtlib/set-info-status.smt2]: Done [2c54794](https://github.com/cvc5/cvc5/pull/8268/commits/2c54794a4ed28b32303d71dc5148ea829bf0602b)

File: test/regress/run_regression.py [hunk 2]
```diff
@@ -470,6 +470,8 @@ def run_benchmark(benchmark_info):
         error = error.decode()
     output = re.sub(r"^[ \t]*", "", output.strip(), flags=re.MULTILINE)
     error = re.sub(r"^[ \t]*", "", error.strip(), flags=re.MULTILINE)
+    error = re.sub(r"qemu: uncaught target signal.*", "", error.strip(),
+                   flags=re.MULTILINE)
```
Comment (nafur) [inline test/regress/run_regression.py:474]:
```suggestion
    # qemu (used for arm nightlies) emits additional error output for non-zero exit codes
    error = re.sub(r"qemu: uncaught target signal.*", "", error.strip())
```

**Review by nafur — APPROVED** [review]
LGTM.

**Pull request merged**